desktop/app-handlers/logging/ipc-handler/index.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+/**
+ * External Dependencies
+ */
+const { ipcMain: ipc } = require( 'electron' );
+
+module.exports = {
+    /**
+     * JSON object that stores references to loggers initialized
+     * by calls from the renderer process over IPC (keyed by `namespace`).
+     */
+	_loggers: {},
```

Comment:
quick note: we don't use the underscore to indicate private methods. [Coding guidelines](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/javascript.md)

It's not entirely or clearly spelled out, but in the React component section it suggests that
we avoid method names prefixed with `_`. In general we don't use too many private methods
at all.

Reply:
Thanks for the tip, definitely something I was unaware of. Updated in 6649f35.